pages/dao/[symbol]/proposal/components/instructions/Pyth/common.ts

```diff
@@ -0,0 +1,5 @@
+import { PublicKey } from '@solana/web3.js'
+
+export const PYTH_STAKING_PROGRAM_ID = new PublicKey(
+  'pytS9TjG1qyAZypk7n8rw8gfW9sUaqqYyMhJQ4E7JCQ'
+)
```

Comment:
You can import it from @pythnetwork/staking

Reply:
Fixed